images/build.sh

```diff
@@ -58,6 +65,9 @@ if [[ "${BUILD_CRUISE}" == "true" ]]; then
   docker image build --build-arg CRUISE_CONTROL_VERSION=${CRUISE_CONTROL_VERSION} --build-arg CRUISE_CONTROL_UI_VERSION=${CRUISE_CONTROL_UI_VERSION} \
     -t mesosphere/cruise-control:${CRUISE_CONTROL_TAG_VERSION} ./cruise-control
 fi
+if [[ "${BUILD_SCALE}" == "true" ]]; then
+  docker image build -t mesosphere/kafka:workload ./scaletesting
```

Comment:
Why `kafka:workload`, shouldn't we capture the version of Kafka instead, i.e. name this `kafka-workload:2.5.1-1.3.1`? With that we could handle the case of running the scale testing with different versions of Kafka.

Reply:
I will ask for a new docker repository with the name of `kafka-workload`

Reply:
done!